src/App.js

```diff
@@ -0,0 +1,23 @@
+import './App.css';
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import HeroSection from './components/hero/HeroSection.js';
+import Header from './components/header/Header.js';
+import SearchField from './components/searchSection/SearchField.js';
+import PollutionData from './components/pollutionDataPage/PollutionData.js';
+
+function App() {
```

Comment:
- [ ] Kindly make sure that the app works correctly. Currently, the app produces some errors when it's started up.

![image](https://user-images.githubusercontent.com/62290545/219445264-c8a03e49-63b6-4e71-99b3-ec09a3d429b6.png)

Reply:
Done 👌

Reply:
resolved! ✔️